tests/unit/segments/test_segments_evaluator.py

```diff
@@ -135,3 +144,31 @@ def test_identity_in_segment_percentage_split(
 
     # Then
     assert result == expected_result
+
+
+@pytest.mark.parametrize(
+    "operator, value, property_, expected_result",
+    (
+        (IS_SET, segment_condition_string_value, segment_condition_property, True),
```

Comment:
I'm not sure why we're giving the segment condition a value here? For IS_SET / IS_NOT_SET segment conditions will not have a value.